test/xpu_api/numerics/complex.number/complex.transcendentals/acos.pass.cpp

```diff
@@ -54,12 +54,12 @@ void test_edges()
         }
         else if (std::isfinite(testcases[i].real()) && std::isinf(testcases[i].imag()))
         {
-#if !_PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL_GLIB_CXX      // testcases[45]
+#if !_PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL               // testcases[45]
             assert(is_about(r.real(), pi/2));
-#endif // _PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL_GLIB_CXX
-#if !_PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL_GLIB_CXX      // testcases[147]
+#endif
+#if !_PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL               // testcases[147]
```

Comment:
Can we remove these two lines, and just protect this with one guard?
Or do we separate them for some specific reason

Reply:
Sure, done.